README.md

```diff
@@ -56,7 +56,7 @@ Alternatively, you can clone the source from GitHub to a directory of Your choic
     sudo python setup.py install 
     
 This will install Fail2Ban into the python library directory. The executable
-scripts are placed into `/usr/bin`, and configuration in `/etc/fail2ban`.
+scripts are placed into `/usr/local/bin`, and configuration in `/etc/fail2ban`.
```

Comment:
Hmm... this depends on distribution-version resp. platform where you try to install fail2ban.
I don't think it's correct so... Maybe:
```diff
- scripts are placed into `/usr/bin`, and configuration in `/etc/fail2ban`.
+ scripts are placed into `/usr/bin` or `/usr/local/bin`, and configuration in `/etc/fail2ban`.
```

Reply:
I change that because the manual install but the fail2ban into `/usr/local/bin` (where distrib installer usually use `/usr/bin`
But we can add both.

Reply:
or `/usr/sbin` or `/usr/local/sbin` since admin-oriented... so may be "scripts are placed in a directory listed in the `PATH` environment variable (e.g., `/usr/bin`), ..."